go.mod

```diff
@@ -1,10 +1,130 @@
 module github.com/openshift/osd-cluster-ready
 
-go 1.15
+go 1.17
```

Comment:
So this matches what boilerplate is doing to your dockerfile etc., but makes me wonder whether boilerplate ought to enforce such.

Reply:
Yeah I think it would be nice - I don't fully grasp the whole picture and if there's reasoning for it, but I've seen consumers of golang-osd-operator range from Go 1.13 to 1.17. If the flexibility isn't desired it's something that could be straightforward to standardize.

To the test case, agree it's contrived/the underlying function is simple haha, but it does exercise the function at least...